doc/pandoc-syntax.txt

```diff
@@ -77,6 +77,10 @@ CONFIGURATION				       *vim-pandoc-syntax-configuration*
 + *g:pandoc#syntax#style#underline_special*
   Undeline subscript, superscript and strikeout text styles. Default = 1
 
++ *g:pandoc#syntax#style#use_definition_lists*
+  Detect and highlight definition lists. Looks really nice, but can result in
+  slower performance. Default = 1 (i.e., on by default)
```

Comment:
Let's not editorialize by criticizing ourselves! ;) 'Disabling this can improve performance' would be enough.